experimental/benchmark/phoronix/scripts/windows-runner.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Run phoronix on windows system via SSH
+
+set -o pipefail
+
+DIR="$(readlink -f "$(dirname "$0")")"
+
+set -x
+
+#PORT=8022
+#HOST=127.0.0.1
+#KEY="${DIR}/../id_rsa"
```

Comment:
Hi, @sarahn. Can you please put private part of `phoronix.pub` somewhere? Or we need to use some kind of cloud-init iso?
We can use `Passw0rd!`, but this variant is interactive.

Also, I try to use `ssh-copy-id` without success:
```
giggsoff@giggsoff-PC:/data/sarah$ ssh-copy-id -i ~/.ssh/id_rsa -p 8022 IEUser@127.0.0.1
/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/giggsoff/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed
/usr/bin/ssh-copy-id: INFO: 1 key(s) remain to be installed -- if you are prompted now it is to install the new keys
IEUser@127.0.0.1's password: 
exec : The term 'exec' is not recognized as the name of a cmdlet, function, script file, or operable program. Check 
the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ exec sh -c 'cd ; umask 077 ; mkdir -p .ssh && { [ -z `tail -1c .ssh/a ...
+ ~~~~
    + CategoryInfo          : ObjectNotFound: (exec:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

Reply:
The playbook produces an image that runs cloud-init at next boot. I assumed you would use cloud-init to add keys to the IEUser user.  If you don't have cloud-init, it's OK to just make a new key pair and modify the playbook with the new key pair.

Reply:
I'm not surprised ssh-copy-id does not work. The shell in the windows VM is powershell, not bash.

Reply:
I can email you the private key if that's what you really need but the existing key was mostly there for me to debug if cloud-init wasn't working.